### PR TITLE
added subject docs on allowing all users

### PR DESCRIPTION
**Pull request by jsloyer**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1839)

### CODE REVIEW

File: docs/admin/authorization.md [hunk 1]
```diff
@@ -440,6 +440,36 @@ subjects:
   name: system:serviceaccounts
 ```
 
+For all users eveywhere with Kube < 1.5:
+```yaml
+subjects:
+- kind: User
+  name: "*"
```
Comment (liggitt) [inline docs/admin/authorization.md]: `name: "*"` doesn't match all users in RBAC

File: docs/admin/authorization.md [hunk 2]
```diff
@@ -440,6 +440,36 @@ subjects:
...
+```
+
+For all authenticated users with Kube > 1.5:
```
Comment (liggitt) [inline docs/admin/authorization.md]: documentation is versioned, I wouldn't version-qualify this bit

File: docs/admin/authorization.md [hunk 3]
```diff
@@ -440,6 +440,36 @@ subjects:
...
+```
+
+For all unauthenticated users with Kube > 1.5:
+```yaml
+subjects:
+- kind: Group
+  name: system:unauthenticated
+```
+
+For all users eveywhere with Kube > 1.5:
```
Comment (liggitt) [inline docs/admin/authorization.md]: typo

**Comment by jsloyer** [comment]
@liggitt I updated the PR based on your review comments

File: docs/admin/authorization.md [hunk 4]
```diff
@@ -440,6 +440,29 @@ subjects:
   name: system:serviceaccounts
 ```
 
+For all authenticated users with Kube:
```
Comment (liggitt) [inline docs/admin/authorization.md]: no need to specify "with Kube"

File: docs/admin/authorization.md [hunk 5]
```diff
@@ -440,6 +440,29 @@ subjects:
...
+```yaml
+subjects:
+- kind: Group
+  name: system:authenticated
+```
+
+For all unauthenticated users with Kube:
```
Comment (liggitt) [inline docs/admin/authorization.md]: no need to specify "with Kube"

**Comment by jsloyer** [comment]
@liggitt made updates based on your review...

**Comment by liggitt** [comment]
squash then LGTM

**Comment by jsloyer** [comment]
@liggitt changes squashed

**Comment by liggitt** [comment]
LGTM, applies to 1.5

**Comment by jaredbhatti** [comment]
If this is a 1.5 release PR, can you please rebase this against the release-1.5 branch?

**Comment by jsloyer** [comment]
@jaredbhatti I rebased on `release-1.5`

**Comment by jsloyer** [comment]
I signed it!

**Comment by liggitt** [comment]
I think he meant open the PR against the 1.5 branch

**Comment by jsloyer** [comment]
closing in favor of https://github.com/kubernetes/kubernetes.github.io/pull/1891 which is based on `release-1.5`.

**Pull request closed without merging**